src/transformers/modeling_bart.py

```diff
@@ -1089,7 +1089,9 @@ def forward(
             past_key_values = unused.pop("decoder_past_key_values")
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
+        assert -100 not in input_ids
```

Comment:
Does this work on a 2D tensor? also why input_ids, shouldnt it just be labels?
Also we should have an error message.

Reply:
```suggestion
```

Reply:
You are right, adding it to the labels was a mistake. It works for me on a 2D tensor, but the other way is also fine.